Source/WebCore/accessibility/AXCoreObject.h

```diff
@@ -1709,7 +1709,7 @@ inline DidTimeout performFunctionOnMainThreadAndWaitWithTimeout(U&& lambda, Seco
     return completedInTime ? DidTimeout::No : DidTimeout::Yes;
 }
 
-template<typename T, typename U> inline T retrieveValueFromMainThread(U&& lambda)
+template<typename T, typename U> inline T retrieveValueFromMainThread(NOESCAPE U&& lambda)
```

Comment:
Is it correct use `NOESCAPE` here, since it appears that the lambda may not be called synchronously?